tools/check-git-version.js

```diff
@@ -0,0 +1,25 @@
+const shell = require('shelljs');
+
+function checkGitVersion() {
+  try {
+    const gitVersion = shell
+      .exec('git --version', { silent: true })
+      .stdout.toString()
+      .slice(12)
```

Comment:
Prefer no magic numbers

Reply:
Can I use a regex to retrieve the version numbers from the string?
Thinking of using this one ```/[\d]+(?=.)/g```  it returns the major, minor, and revision version number in an array and then compares to GIT_MINIMUM_VERSION.

Reply:
It just needs to work reliably

Reply:
I think simple-git can do this parsing for us more easy. Have you tested it?

Reply:
I could not find any command in simple-git to do the parsing for us.
Also, I was wondering if I can use the npm-package ```compare-versions``.

Reply:
You can use semver package. You first need to coerse, then you can do simple comparison.

Reply:
If simple-git didn't have a prepared function use shelljs. But better use a regex to extract the version.

Reply:
I did it using regex and semver.